beard/metrics/clustering.py

```diff
@@ -19,9 +20,23 @@
 from operator import mul
 from itertools import groupby
 
+from sklearn.metrics import silhouette_score
 from sklearn.metrics.cluster.supervised import check_clusterings
 
 
+def silhouette(X, labels, metric="precomputed"):
```

Comment:
Please keep the `_score` suffix. And to avoid name clashes, import the function from sklearn using
`from sklearn.metrics import silhouette_score as sklearn_silhouette_score`